ui/nuxeo-video/nuxeo-video-conversions.js

```diff
@@ -0,0 +1,93 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import '@polymer/iron-flex-layout/iron-flex-layout-classes.js';
+import { FormatBehavior } from '../nuxeo-format-behavior.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
+import '../nuxeo-icons.js';
+import '../widgets/nuxeo-tooltip.js';
+
+{
+  /**
+  `nuxeo-video-conversions`
+  @group Nuxeo UI
+  @element nuxeo-video-conversions
+  */
+  class VideoConversions extends mixinBehaviors([I18nBehavior, FormatBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style include="iron-flex">
+          a,
+          a:active,
+          a:visited,
+          a:focus {
+            @apply --nuxeo-link;
+          }
+          a:hover {
+            @apply --nuxeo-link-hover;
+          }
+          #flexContainer {
+            @apply --layout-flex;
+            @apply --layout-horizontal;
+            @apply --layout-center;
+          }
+        </style>
+        <dom-if if="[[label]]">
+          <template>
+            <h3>[[label]]</h3>
+          </template>
+        </dom-if>
+        <div>
+          <dom-repeat items="[[document.properties.vid:transcodedVideos]]" as="conversion">
+            <dom-if if="[[conversion.content]]">
+              <template>
+                <div id="flexContainer">
+                  <label class="flex">[[conversion.name]]</label>
```

Comment:
Since we are already using the mixins above we can ditch the iron flex classes. We could create our own class with a better meaning, let's say "conversion-detail" or just "detail" or "item" (or something else, be creative), and set `@apply --layout-flex;` on it. This way, we could remove the dependency on the `iron-flex` styles module and remove the import for `iron-flex-layout-classes.js`.